db/migrate/20220811013101_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: { to_table: 'users' }
+      t.string :title
+      t.string :text
```

Comment:
- [ ] Here also it would be better to use `t.text` instead of `t.string` for `:text`.